modules/world/world.cpp

```diff
@@ -123,7 +123,7 @@ EvaluationMap World::Evaluate() const {
 }
 
 std::vector<ObservedWorld> World::Observe(
-    const std::vector<AgentId>& agent_ids) {
+    const std::vector<AgentId>& agent_ids) const {
```

Comment:
Do not use const here. otherwise e.g. a PredictionSetting cannot be set up.
The behavior of other vehicles should still be changable.